documentation/pdb_client_tools.markdown

```diff
@@ -48,11 +55,49 @@ certificate-whitelist and specify the paths to the CLI node's cacert, cert, and
 private key when using the CLI either with flags or a configuration file.
 
 To configure the PuppetDB CLI to talk to your PuppetDB with flags, add a
-configuration file at `$HOME/.puppetlabs/client-tools/puppetdb.conf`. For more
+configuration file at `$HOME/.puppetlabs/client-tools/puppetdb.conf` (or
+`$USERPROFILE/.puppetlabs/client-tools/puppetdb.conf` for Windows). For more
 details see the installed man page:
 
     $ man puppetdb_conf
 
+The PuppetDB CLI configuration files (the user-specified or global files) can
+take the following settings:
+
+- `server_urls` Either a JSON String (for a single url) or Array (for multiple
+  urls) of your PuppetDB servers to query or manage via the CLI commands. (You
+  can set this with the `puppetdb_urls` parameter in the
+  puppet_enterprise::profile::controller class for PE.)
+
+  Default value: https://127.0.0.1:8080
+
+- `cacert` The path for the CA cert.
+
+  *nix sytems - /etc/puppetlabs/puppet/ssl/certs/ca.pem  
+  Windows - C:/ProgramData/PuppetLabs/client-tools/ssl/certs/ca.pem
```

Comment:
@ajroetker, sorry, but upon further review what I told you was wrong. take out the unmanaged and managed and just say this is the path to the ca cert: `C:\ProgramData\PuppetLabs\puppet\etc\ssl\certs\ca.pem`. I asked Nick Lewis to check the code.